lib/webpacker_uploader/providers/aws.rb

```diff
@@ -25,6 +25,8 @@ def upload!(object_key, file, content_type = "")
         def credentials(options)
           if options[:profile_name].present?
             ::Aws::SharedCredentials.new(profile_name: options[:profile_name])
+          elsif options.key?(:from_ec2) && options[:from_ec2]
```

Comment:
Would it make sense to rename the key from `from_ec2` to `instance_profile`? I can also rename the profile option above from `profile_name` to `aws_profile` to be more descriptive and consistent.

Also, I think the second check is redundant here.

Reply:
I'd do the change. Also, I'll keep the `profile_name` key and add a `aws_profile` key instead, to keep both options available.